plugin/storage/badger/samplingstore/storage.go

```diff
@@ -96,7 +96,7 @@ func (s *SamplingStore) GetThroughput(start, end time.Time) ([]*model.Throughput
 				return err
 			}
 
-			if t.After(start) && (t.Before(end) || t.Equal(end)) {
+			if t.After(start) && (t.Before(end) || t.Equal(end)) && len(throughputs) > 0 {
```

Comment:
this doesn't change anything, append works just fine with nil slice

Reply:
@yurishkuro I updated.